resources/views/dashboard/controllers/files.blade.php

```diff
@@ -21,446 +21,103 @@
-										@endif
-										@if(!$loop->last)
-											<a onclick="itemReorder({{ $f->id }},{{ $loop->index }},{{ $f->type }},'down');" class="btn btn-info simple-tooltip" data-toggle="tooltip" title="Down"><i class="fas fa-arrow-down"></i></a>
-										@endif
-                                    @endif
-								</div>
-                                </td>
-                            @endif
-                            </tr>
-                        @endforeach
-                    @endif
-                </tbody>
-            </table>
-        </div>
-        <div role="tabpanel" class="tab-pane" id="staff">
-            <table class="table table-bordered table-striped">
-                <thead>
-                    <tr>
-                        <th scope="col">Name</th>
-                        <th scope="col"><center>Description</center></th>
-                        <th scope="col"><center>Uploaded/Updated at</center></th>
-                        <th scope="col"><center>Actions</center></th>
-                    </tr>
-                </thead>
-                <tbody>
-                    @if($staff->count() > 0)
-                        @foreach($staff as $f)
-                            <tr>
-                            @if($f->row_separator)
-                                @if(Auth::user()->isAbleTo('files'))
-                                    <th class="text-center" colspan="3">{{ $f->name }}</th>
-                                    <td>
-                                        <a href="/dashboard/admin/files/edit/{{ $f->id }}" class="btn btn-warning simple-tooltip" data-toggle="tooltip" title="Edit"><i class="fas fa-pencil-alt"></i></a>
-                                        <a href="/dashboard/admin/files/delete/{{ $f->id }}" onclick="return confirm('Are you sure you want to delete {{ $f->name }}?')" class="btn btn-danger simple-tooltip" data-toggle="tooltip" title="Delete"><i class="fas fa-times"></i></a>
-                                        @if(!$loop->first)
-      										<a onclick="itemReorder({{ $f->id }},{{ $loop->index }},{{ $f->type }},'up');" class="btn btn-info simple-tooltip" data-toggle="tooltip" title="Up"><i class="fas fa-arrow-up"></i></a>
-	    								@endif
-		    							@if(!$loop->last)
-			    							<a onclick="itemReorder({{ $f->id }},{{ $loop->index }},{{ $f->type }},'down');" class="btn btn-info simple-tooltip" data-toggle="tooltip" title="Down"><i class="fas fa-arrow-down"></i></a>
-				    					@endif
-                                    </td>
                                 @else
-                                    <th class="text-center" colspan="4">{{ $f->name }}</th>
+                                    <td>{{ $f->name }}</td>
+                                    <td>{{ $f->desc }}</td>
+                                    <td>{{ $f->updated_at }}</td>
+                                    <td>
+	    			    				<div class="btn-group">
```

Comment:
See above comment about icons (add `fa-fw` to all of these icons)

Reply:
Done!

Reply:
Looks great. Thx